tests/keras/test_keras_model_export.py

```diff
@@ -67,32 +69,37 @@ def keras_custom_env(tmpdir):
 
 
 def test_model_save_load(model, model_path, data, predicted):
-    x, y = data
-    mlflow.keras.save_model(model, model_path)
+    with TempDir() as dir:
```

Comment:
Sorry, not sure I understand the intention of the change -- isn't `model_path` already a tmpdir?

Reply:
```model_path``` is reused across a couple of tests. Most of the time it works but sometimes this test is failing at random.

Here are examples:

https://travis-ci.org/mlflow/mlflow/jobs/510161501
https://travis-ci.org/mlflow/mlflow/jobs/509371870

the only reason I see it might fail is shared tmp directory.

Reply:
@aarondav is this sufficient explanation?

Reply:
@aarondav ping. is this sufficient explanation? or should I close this PR?

Reply:
Hi @jaroslawk, thanks for bringing this up - I believe the temp directory is actually not shared across tests - `model_path` is defined as a `@pytest.fixture` with no scope, which according to the pytest docs ([link](https://docs.pytest.org/en/latest/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session)) means it'll by default be invoked per test function

Reply:
Ok I see your point. Closing it now then.